src/bin/evtx_dump.rs

```diff
@@ -174,6 +182,16 @@ fn main() {
                 .takes_value(false)
                 .help("When set, `Record <id>` will not be printed."),
         )
+        .arg(
+            Arg::with_name("ansi-codec")
+                .long("--ansi-codec")
+                .possible_values(&encodings().iter()
+                    .filter(|&e| e.raw_decoder().is_ascii_compatible())
+                    .map(|e| e.name())
+                    .collect::<Vec<&'static str>>())
+                .default_value(encoding::all::WINDOWS_1252.name())
+                .help("When set, controls the codec of ansi encoded strings the file. defaults to `WINDOWS-1252`"),
```

Comment:
Put a link to the docs where all the encodings are defined?

Reply:
It prints it nicely if you use `evtx_dump --help`